src/Files.App/Services/Windows/WindowsIniService.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Services
+{
+	/// <inheritdoc cref="IWindowsIniService"/>
+	public sealed class WindowsIniService : IWindowsIniService
+	{
+		/// <inheritdoc/>
+		public List<IniSectionDataItem> GetData(string filePath)
+		{
+			var iniPath = SystemIO.Path.Combine(filePath);
+			if (SystemIO.File.Exists(iniPath) is false)
+				return [];
+
+			var lines = SystemIO.File.ReadLines(iniPath)
```

Comment:
We need to consider the case where the file exists but cannot be accessed. See #15732